CONTRIBUTING.md

```diff
@@ -1,155 +0,0 @@
-# Contributing to Chef
```

Comment:
Just nuke the file and toss a contributing section with the link in the readme. Keeps it simple